generators/app/templates/src/main/docker/akhq.yml.ejs

```diff
@@ -0,0 +1,15 @@
+version: '<%= dockerComposeFormatVersion %>'
+services:
+  akhq:
+    image: <%= dockerAkhq %>
```

Comment:
I can see in other file:

```
this.dockerAkhq = 'tchiotludo/akhq';
```

It's better to use a tag, instead of the latest version